### PR TITLE
Version Page Added

### PR DESCRIPTION
**Pull request by jimil749**
Frontend Changes for cBioPortal/cbioportal#7025

Note that this PR does not contain the data version and the External services.

Describe changes proposed in this pull request:
- a add a seperate page on the frontend that describes the software version in more detail.

versionPage

# Notify reviewers
@inodb

### CODE REVIEW

**Comment by ecerami** [comment]
Thanks @jimil749!  Is it ok if we add the DO NOT MERGE tag until you have added the live data version info?

File: src/pages/staticPages/version/Version.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import AppConfig from 'appConfig';
+import StaticContent from '../../../shared/components/staticContent/StaticContent';
+import Helmet from 'react-helmet';
+
+export default class Version extends React.Component<{}, {}> {
+    public render() {
+        return (
+            <PageLayout className="whiteBackground staticPage">
+                <Helmet>
+                    <title>{'cBioPortal for Cancer Genomics::Version'}</title>
```
Comment (inodb) [inline src/pages/staticPages/version/Version.tsx]: Maybe `cBioPortal for Cancer Genomics Version`? The `::` looks a bit too technical

**Comment by inodb** [comment]
@jimil749 Instead of adding `Version` to the heading section. I think it would be enough to add it as a link from the footer, so e.g. `v3.2.2` is a link to the page you have created. For most people the version is probably not that important to access quickly that it warrants a link in the header

As a side note: it is fine that the end to end tests fail here. We are working on fixing those, it's unrelated to your code

File: src/pages/staticPages/version/Version.tsx (hunk 1 above)
Comment (jimil749) [reply src/pages/staticPages/version/Version.tsx]: Sure, will change that!

**Comment by jimil749** [comment]
Thank you for the feedback. I will start working on the requested changes.

**Comment by inodb** [comment]
Hi @jimil749 - thanks so much for your work! We found out today we weren't accepted for GSoC2020. We would love to accept further contributions from you, but understand if you will look for another open source org to contribute. Thank you!

**Comment by jimil749** [comment]
I would still like to contribute for the organization, at least complete the task assigned to me right now! :)

**Comment by inodb** [comment]
Great! Thank you @jimil749 🙏!

**Pull request closed without merging**